src/test/java/dawahdevs/qa/api/managerTest/QaApiManagerTest.java

```diff
@@ -6,24 +6,33 @@
 import org.junit.Test;
 import org.mockito.InjectMocks;
 
-import dawahdevs.qa.api.manager.QuestionManager;
+import dawahdevs.qa.api.manager.QaApiManager;
 
 public class QaApiManagerTest {
 
+//	@Mock
+//	private final QaApiTransformer qaApiTransformer;
+//
+//	@Mock
+//	private final QuestionRepository questionRepository;
+//
+//	@Mock
+//	private final UserRepository userRepository;
+//
+//	@Mock
+//	private final AuthorRepository authorRepository;
+
 	@InjectMocks
-	private QuestionManager qaApiManager;
+	private QaApiManager qaApiManager;
 
 	@Before
 	public void init() {
 		initMocks(this);
-	}
 
-	@Test
-	public void testGetQuestions() {
 	}
 
 	@Test
-	public void testGetQuestion() {
+	public void testGetQuestions() {
 	}
 
 }
```

Comment:
In fact remove this entire test file since its empty. Dont add code unless it is needed now. This just adds more irrelevant code to maintain. Add future code in the future.